desc/coils.py

```diff
@@ -224,7 +224,7 @@ def compute_magnetic_field(
             current = params.pop("current", self.current)
 
         data = self.compute(
-            ["x", "x_s", "ds"], grid=source_grid, params=params, basis="xyz"
+            ["phi", "x", "x_s", "ds"], grid=source_grid, params=params, basis="xyz"
```

Comment:
do we need to add phi here? is this because of the issue with adding it automatically in the compute func?

Reply:
@ddudt added that one

Reply:
Yes I added this before we added the logic to automatically add phi when needed. I think this might still be necessary because computing `"phi"` can require additional `params` relative to the other compute quantities for some Curve classes. See my other comment about this